test/Kestrel.FunctionalTests/RequestBodyTimeoutTests.cs

```diff
@@ -182,9 +189,9 @@ public class RequestBodyTimeoutTests : LoggedTest
                         "",
                         "");
 
-                    Assert.True(appRunningEvent.Wait(TestConstants.DefaultTimeout), "AppRunningEvent timed out.");
+                    await appRunningTcs.Task.DefaultTimeout();
                     systemClock.UtcNow += gracePeriod + TimeSpan.FromSeconds(1);
-                    Assert.True(exceptionSwallowedEvent.Wait(TestConstants.DefaultTimeout), "ExceptionSwallowedEvent timed out.");
+                    await exceptionSwallowedTcs.Task.DefaultTimeout();
```

Comment:
What are we asserting now? If there's nothing that we should/can assert I generally like to leave a comment noting the exception we're confirming we don't it, or something along those lines. That helps make it clear that we didn't just forget the Assert (which I've seen in our code-base), it was an active choice.

Reply:
DefaultTimeout() is an assertion. This is common in Kestrel functional tests. Another common assertion is awaiting a TCS where SetException is set in the app func. Adding comments everywhere pointing out this is an assertion would be super noisy.

Reply:
Same goes for Receive[ForcedEnd]. That's also an assertion that we use everywhere. Anyone who works at all with Kestrel functional will be familiar with this convention.

Reply:
Cool beans.